components/page/content-front-page-panels.php

```diff
@@ -47,12 +44,38 @@
 				<?php
 					/* translators: %s: Name of current post */
 					the_content( sprintf(
-						__( 'Continue reading %s <span class="meta-nav">&rarr;</span>', 'twentyseventeen' ),
+						__( 'Continue reading %s', 'twentyseventeen' ),
 						the_title( '<span class="screen-reader-text">"', '"</span>', false )
 					) );
 				?>
 			</div><!-- .entry-content -->
 
+			<?php
+			// Show recent blog posts if is blog posts page (Note that get_option returns a string, so we're casting the result as an int)
+			if ( get_the_ID() === (int) get_option( 'page_for_posts' )  ) : ?>
+
+				<?php // Show four most recent posts
+				$recent_posts = new WP_Query( array(
+					'posts_per_page'      => 3,
+					'post_status'         => 'publish',
+					'ignore_sticky_posts' => true
```

Comment:
As there is no pagination we should be able to add `'no_found_rows' => true` which will improve the query performance. https://10up.github.io/Engineering-Best-Practices/php/#performance